src/deps.ts

```diff
@@ -141,6 +131,7 @@ interface AWS4 {
    * @param credentials - AWS credential details, sessionToken should be omitted entirely if its false-y
    */
   sign(
+    this: void,
```

Comment:
Removing this throws an error in mongodb_aws.ts, but I actually think that binding the function in mongodb_aws is the correct solution, not adding this to the type.

```typescript
   const sign = aws4.sign.bind(aws4);
```

thoughts?